core-shaded/pom.xml

```diff
@@ -311,6 +300,11 @@
                   <artifactId>jctools-core</artifactId>
                   <version>2.1.2</version>
                 </additionalDependency>
+                <additionalDependency>
+                  <groupId>com.esri.geometry</groupId>
+                  <artifactId>esri-geometry-api</artifactId>
+                  <version>1.2.1</version>
+                </additionalDependency>
```

Comment:
javadoc generation was throwing warnings about missing classes without this declaration in place